codegenerator/cli/templates/static/codegen/src/EventProcessing.res

```diff
@@ -298,45 +296,47 @@ let runEventHandler = (
           ~decimalPlaces=4,
         )
       }
-      eventBatchQueueItem->updateEventSyncState(
-        ~inMemoryStore,
-        ~isPreRegisteringDynamicContracts=false,
-      )
-      if config.enableRawEvents {
-        eventBatchQueueItem->addEventToRawEvents(~inMemoryStore)
-      }
-      latestProcessedBlocks
-      ->EventsProcessed.updateEventsProcessed(
-        ~chain=eventBatchQueueItem.chain,
-        ~blockNumber=eventBatchQueueItem.blockNumber,
-      )
-      ->Ok
```

Comment:
This get's moved lower down the call stack so that it's run whether or not a handler is registered